code/game/gamemodes/endgame/supermatter_cascade/universe.dm

```diff
@@ -87,6 +92,16 @@
 			APC.emagged = 1
 			APC.queue_icon_update()
 
+			APC.areaMaster.power_light = 0
+			APC.areaMaster.power_equip = 0
+			APC.areaMaster.power_environ = 0
+			APC.areaMaster.power_change()
```

Comment:
What the fuck did you do deity. Remove this line. Seriously, NEVER use this proc.